src/test/scala/lenthall/spray/WrappedRouteSpec.scala

```diff
@@ -0,0 +1,87 @@
+package lenthall.spray
+
+import lenthall.spray.WrappedRoute._
+import org.scalatest.{FlatSpec, Matchers}
+import spray.http.StatusCodes
+import spray.routing.HttpService
+import spray.testkit.ScalatestRouteTest
+
+class WrappedRouteSpec extends FlatSpec with Matchers with HttpService with ScalatestRouteTest {
+  override def actorRefFactory = system
+
+  def unwrappedRoute = path("hello") {
+    get {
+      complete("in get")
+    } ~ post {
+      complete("in post")
+    }
+  }
+
+  behavior of "WrappedRoute"
+
+  it should "service wrapped routes" in {
+    Get("/api/hello") ~> unwrappedRoute.wrapped("api") ~> check {
```

Comment:
The `unwrappedRoute.wrapped("api")` may deserve its own `val wrappedRoute = unwrappedRoute.wrapped("api")` definition as it's used a bunch of times.
Same for `val legacyWrappedRoute = unwrappedRoute.wrapped("api", routeUnwrapped = true)`